ApnsPHP/Message.php

```diff
@@ -528,14 +528,9 @@ public function getPayload()
             defined('JSON_UNESCAPED_UNICODE') ? JSON_UNESCAPED_UNICODE : 0
         );
         if (!defined('JSON_UNESCAPED_UNICODE') && function_exists('mb_convert_encoding')) {
-            $JSON = preg_replace_callback(
-                '~\\\\u([0-9a-f]{4})~i',
-                create_function(
-                    '$aMatches',
-                    'return mb_convert_encoding(pack("H*", $aMatches[1]), "UTF-8", "UTF-16");'
-                ),
-                $JSON
-            );
+            $JSON = preg_replace_callback('~\\\\u([0-9a-f]{4})~i', function ($aMatches) {
+                return mb_convert_encoding(pack("H*", $aMatches[1]), "UTF-8", "UTF-16");
```

Comment:
```suggestion
            $JSON = preg_replace_callback('~\\\\u([0-9a-f]{4})~i', function ($aMatches) {
                return mb_convert_encoding(pack("H*", $aMatches[1]), "UTF-8", "UTF-16");
```
I renamed all variables so they don't have the type in their name in a previous commit, but I must have missed this one because it was in a string. Makes things more consistent.